how-to/customize-workspace/client/src/framework/workspace/home.ts

```diff
@@ -382,7 +381,7 @@ async function getResults(
 	};
 }
 
-export async function register(): Promise<RegistrationMetaInfo> {
+export async function register(): Promise<HomeRegistration> {
```

Comment:
Is this a breaking change (at compile time TS will pick this up but has the shape of the object changed at all?)

Reply:
As I suggested in the PRD HomeRegistration is derived from RegistrationMetaInfo but with the additional setSearchQuery method